### DISCUSSION TITLE
Welcome to ESPRotary Discussions!

### DISCUSSION THREAD

**Discussion by LennartHennigs** [LennartHennigs/ESPRotary#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WJurasek-KV** [comment]
@LennartHennigs Hey, I just pulled your code and went to use it! It seems to be broken right now though. Looking through your last commit, you renamed some variables and it led to situations like: 

void ESPRotary::setIncrement(int increment) {
  increment = increment;
}

I don't know if this was intentional but the code only returns 0's now

**Reply by LennartHennigs** [reply]
Hey, 
thank you for pointing this out!
I forgot a `this->`...
Just pushed an update.

Cheers
l.

**Reply by WJurasek-KV** [reply]
Works great now!
